app/src/main/java/com/vero/tasky/agenda/presentation/agendaevent/components/PhotosList.kt

```diff
@@ -54,14 +56,23 @@ fun PhotoList(
                 color = MaterialTheme.colors.primary,
                 style = MaterialTheme.typography.Inter600Size20
             )
+            Spacer(modifier = Modifier.height(8.dp))
             LazyVerticalGrid(
-                columns = GridCells.Adaptive(60.dp)
+                columns = GridCells.Adaptive(68.dp)
```

Comment:
Why are you using a VerticalGrid for the photos? Arent the photos displayed in a row?
![image](https://github.com/AndreVero/Tasky/assets/16141845/6d8b8db9-3280-49ff-a7c5-4b8bace5f657)

Pretty sure it should be a LazyRow

Reply:
Yeh, it wasn't in the requirements, just more pretty as for me to show it like that (my 5 cents to the app)